src/codecovio.jl

```diff
@@ -225,7 +215,7 @@ module Codecov
 
         uri_str = "$(codecov_url)/upload/v2?"
         for (k,v) in kwargs
-            if k != :codecov_url && k != :dry_run && k != :verbose
```

Comment:
If the user does pass `verbose` as a keyword, removing the explicit handling here will mean it gets added to the URL string.

Reply:
Done, and added a comment